components/brave_extension/extension/brave_extension/manifest.json

```diff
@@ -18,10 +18,15 @@
   },
   "web_accessible_resources": [
     "elementPicker.html",
-    "elementPicker.js"
+    "elementPicker.js",
+    "star.wasm",
+    "modules/hpnv2/worker.asmjs.bundle.js",
+    "modules/hpnv2/worker.wasm.bundle.js"
```

Comment:
Did you try without these here? I still find it surprising that this is needed.

Reply:
yeah. the background script fetches these on load. if i don't have them it throws errors.